python/ray/data/_internal/stats.py

```diff
@@ -776,6 +776,11 @@ def to_summary(self) -> "DatasetStatsSummary":
         stats_summary_parents = []
         if self.parents is not None:
             stats_summary_parents = [p.to_summary() for p in self.parents]
+        streaming_exec_schedule = (
```

Comment:
Nit: It wasn't obvious to me that this variable represents a number based on the name.
```suggestion
        streaming_exec_schedule_s = (
```

Reply:
renamed!